src/server/rdb_load.cc

```diff
@@ -1991,6 +1991,15 @@ RdbLoader::~RdbLoader() {
       break;
     delete item;
   }
+
+  // Decommit local memory.
+  // We create an RdbLoader for each thread, so each one will Decommit for itself after
+  // full sync ends (since we explicitly reset the RdbLoader).
+  auto* tlocal = ServerState::tlocal();
+  if (tlocal) {
+    tlocal->DecommitMemory(ServerState::kDataHeap | ServerState::kBackingHeap |
```

Comment:
How about creating a `ServerState::kAllMemory` to contain all bits?

Reply:
רעיון טוב

Reply:
I think this should mean good idea in Hebrew :rofl: